### ISSUE TITLE
Error in attribute collection / sign up flow page #1220

### ISSUE THREAD

**Issue by kberawala** [AzureAD/microsoft-authentication-library-for-dotnet#2669]
Apologies for writing here - couldnt find the right place to report this.

I have 4 new signups in last 24 hours and havent received lastname from the signup, it show blank in the profile.

I have this - attributes to collect at sign up using the standard policy
![image](https://user-images.githubusercontent.com/13636878/119787147-40a8fd80-bf14-11eb-9eb1-488c09fe4b11.png)

Appears twice - (earlier than today, it was right appeared only once)
![image](https://user-images.githubusercontent.com/13636878/119787386-7ea62180-bf14-11eb-814f-4938115a8265.png)

submitting data with as in screen shot - correctly registers the user
![image](https://user-images.githubusercontent.com/13636878/119787560-ab5a3900-bf14-11eb-8d49-949fc81ad591.png)

ignores the lastname
![image](https://user-images.githubusercontent.com/13636878/119787761-e52b3f80-bf14-11eb-8c09-47003c2bb0a0.png)

**Comment by jmprieur** [comment]
Closing a duplicate of https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2669
